ldms/src/ldmsd/ldmsd_row_cache.c

```diff
@@ -302,6 +302,7 @@ int ldmsd_row_cache(ldmsd_row_cache_t rcache,
 	rc = clock_gettime(CLOCK_REALTIME, &ts);
```

Comment:
@baallan, this may cause some heartburn, but ...

- clock_gettime() with these arguments will never produce an error.
- There are ~117 clock_gettime() call sites in our code.
- Three (3) of them check the return code.

I suggest we remove the return code checking from the three call sites and call it a day.

Reply:
@tom95858 since the 3 places where there are checks are there to control logic rather than just collect data, I suggest we don't remove them without tangible proof that those arguments always work. I didn't manage to track down that proof in a 20 minute dive in glibc, let alone odd libs on odd hpc hardware.